modules/nw-how-nw-iface-selected.adoc

```diff
@@ -20,8 +20,9 @@ As a result, the kubelet service uses the selected IP address as the node IP add
 If hardware or networking is reconfigured after installation, it is possible that the `nodeip-configuration.service` service can select a different NIC after a reboot.
 In some cases, you might be able to detect that a different NIC is selected by reviewing the `INTERNAL-IP` column in the output from the `oc get nodes -o wide` command.
 
-If network communication is disrupted or misconfigured because a different NIC is selected, one strategy for overriding the selection process is to set the correct IP address explicitly.
-The following list identifies the high-level steps and considerations:
+If network communication is disrupted or misconfigured because a different NIC is selected, you can create a hint file that includes the `NODEIP_HINT` variable to override the default IP selection logic. To create a hint file, see Optional: Overriding the default node IP selection logic. 
+
+In some situations, creating a hint file might be insufficient for overriding the selection process. The following list identifies the high-level steps and considerations to manually override the IP address: 
```

Comment:
We still need to remove this and line 27.

Reply:
I am so sorry. I made these changes on the old PR on accident.